guides/deployment_guide.md

```diff
@@ -34,3 +34,98 @@ We use a variety of technologies to get this code into production.  Starting fro
       1. the images with v{semver} will automatically be deployed to prod
 1. The eks clusters in the ess account are configured with fluxcd/helm operator to watch those helm release files
 1. terraform defines the eks clusters, and any databases we may require (https://github.com/mozilla-it/ctms-infra/tree/main/terraform)
+
+
+## Logging
+
+When the environment variable ``CTMS_USE_MOZLOG`` is set to true or unset, then
+the [MozLog JSON format](https://wiki.mozilla.org/Firefox/Services/Logging) is
+used for logging.
+
+## Example
```

Comment:
Minor Nit: maybe could be an h3 instead of h2 header since it's an example of the logging (h2) above? 

_it's just documentation so... 🤷_